service/build.gradle

```diff
@@ -33,19 +33,19 @@ apply from: 'analysis.gradle'
 
 dependencyManagement {
 	dependencies {
-		dependencySet(group:'io.github.hakky54', version: '7.3.0') {
-			entry 'sslcontext-kickstart-for-apache4'
+		dependencySet(group:'io.github.hakky54', version: '8.2.0') {
```

Comment:
This upgrade had the extra benefit of resolving https://broadworkbench.atlassian.net/browse/DR-3391 -- we needed these bouncycastle dependencies >= 1.74.

```
(base) okotsopo@wm111-e35 terra-drs-hub % ./gradlew service:dependencyInsight --dependency bouncycastle

> Task :service:dependencyInsight
org.bouncycastle:bcpkix-jdk18on:1.76
  Variant compile:
    | Attribute Name                 | Provided | Requested    |
    |--------------------------------|----------|--------------|
    | org.gradle.status              | release  |              |
    | org.gradle.category            | library  | library      |
    | org.gradle.libraryelements     | jar      | classes      |
    | org.gradle.usage               | java-api | java-api     |
    | org.gradle.dependency.bundling |          | external     |
    | org.gradle.jvm.environment     |          | standard-jvm |
    | org.gradle.jvm.version         |          | 17           |

org.bouncycastle:bcpkix-jdk18on:1.76
\--- io.github.hakky54:sslcontext-kickstart-for-pem:8.2.0
     \--- compileClasspath (requested io.github.hakky54:sslcontext-kickstart-for-pem)

org.bouncycastle:bcprov-jdk18on:1.76
  Variant compile:
    | Attribute Name                 | Provided | Requested    |
    |--------------------------------|----------|--------------|
    | org.gradle.status              | release  |              |
    | org.gradle.category            | library  | library      |
    | org.gradle.libraryelements     | jar      | classes      |
    | org.gradle.usage               | java-api | java-api     |
    | org.gradle.dependency.bundling |          | external     |
    | org.gradle.jvm.environment     |          | standard-jvm |
    | org.gradle.jvm.version         |          | 17           |

org.bouncycastle:bcprov-jdk18on:1.76
+--- org.bouncycastle:bcpkix-jdk18on:1.76
|    \--- io.github.hakky54:sslcontext-kickstart-for-pem:8.2.0
|         \--- compileClasspath (requested io.github.hakky54:sslcontext-kickstart-for-pem)
\--- org.bouncycastle:bcutil-jdk18on:1.76
     \--- org.bouncycastle:bcpkix-jdk18on:1.76 (*)

org.bouncycastle:bcutil-jdk18on:1.76
  Variant compile:
    | Attribute Name                 | Provided | Requested    |
    |--------------------------------|----------|--------------|
    | org.gradle.status              | release  |              |
    | org.gradle.category            | library  | library      |
    | org.gradle.libraryelements     | jar      | classes      |
    | org.gradle.usage               | java-api | java-api     |
    | org.gradle.dependency.bundling |          | external     |
    | org.gradle.jvm.environment     |          | standard-jvm |
    | org.gradle.jvm.version         |          | 17           |

org.bouncycastle:bcutil-jdk18on:1.76
\--- org.bouncycastle:bcpkix-jdk18on:1.76
     \--- io.github.hakky54:sslcontext-kickstart-for-pem:8.2.0
          \--- compileClasspath (requested io.github.hakky54:sslcontext-kickstart-for-pem)

(*) - dependencies omitted (listed previously)
```